Frends.CassandraDB.Execute/Frends.CassandraDB.Execute/Definitions/Result.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+
+namespace Frends.CassandraDB.Execute.Definitions;
+
+/// <summary>
+/// Task's result.
+/// </summary>
+public class Result
+{
+    /// <summary>
+    /// True if query ran successfully.
+    /// </summary>
+    /// <example>true</example>
+    public bool Success { get; private set; }
+
+    /// <summary>
+    /// Query results. JSON string of results (Only SELECT query will contain values).
+    /// </summary>
+    /// <example>{\"0\":[{\"Key\":\"foo\",\"Value\":\"bar\"}],\"1\":[{\"Key\":\"foo2\",\"Value\":\"bar2\"}]}</example>
+    public string QueryResults { get; private set; }
```

Comment:
You should return `JObject` instead of `string`, since it would enable usage of this data in other Process steps. If user requires JSON string, he can call JObjects `ToString` method.